lib/rules/prefer-regex-literals.js

```diff
@@ -146,6 +146,7 @@ module.exports = {
         messages: {
             unexpectedRegExp: "Use a regular expression literal instead of the 'RegExp' constructor.",
             replaceWithLiteral: "Replace with an equivalent regular expression literal.",
+            replaceWithLiteralAndFlags: "Replace with an equivalent regular expression literal with flags.",
```

Comment:
I think we should add the flags to this message. Otherwise, when there are multiple choices they would look the same.

![image](https://user-images.githubusercontent.com/44349756/208183048-657976b4-498e-4fcf-808a-c5bcf6752513.png)